controllers/cloudstackmachine_controller.go

```diff
@@ -179,6 +186,15 @@ func (r *CloudStackMachineReconciliationRunner) GetOrCreateVMInstance() (retRes
 	return ctrl.Result{}, err
 }
 
+func processUserData(data []byte, r *CloudStackMachineReconciliationRunner) string {
+	// since cloudstack metadata does not allow custom data added into meta_data, following line is a walkaround to specify a hostname name
```

Comment:
```suggestion
	// since cloudstack metadata does not allow custom data added into meta_data, following line is a workaround to specify a hostname name
```